tests/test_wand.py

```diff
@@ -201,11 +201,25 @@ def test_open_webp(self):
                      "imagemagic was not built with WebP support")
     def test_open_webp_w_alpha(self):
         with open('tests/images/tux_w_alpha.webp', 'rb') as f:
-            image = PillowImage.open(WebPImageFile(f))
+            image = WandImage.open(WebPImageFile(f))
 
         self.assertTrue(image.has_alpha())
         self.assertFalse(image.has_animation())
 
+    @unittest.skipIf(no_webp_support,
+                     "Pillow does not have WebP support")
+    def test_open_webp_quality(self):
+        high_quality = self.image.save_as_webp(io.BytesIO(), quality=90)
+        low_quality = self.image.save_as_webp(io.BytesIO(), quality=30)
+        self.assertTrue(low_quality.f.tell() < high_quality.f.tell())
+
+    @unittest.skipIf(no_webp_support,
+                     "Pillow does not have WebP support")
+    def test_open_webp_lossless(self):
```

Comment:
Could we check that the pixel data stays the same?

Reply:
Pixel-wise comparison for Wand takes about 10 seconds. The RGB part of completely transparent pixels is different, so I skipped them.